src-static/update_acled_info.R

```diff
@@ -35,19 +35,19 @@ if (interactive()) {
       )
     ) |>
     dplyr$transmute(
-      iso3 = country_codes$ison_to_iso3(`ISO Codes`),
+      iso3 = location_codes$ison_to_iso3(`ISO Codes`),
       start_date = as.Date(
         x = paste(1, `Start Date`),
         format = "%d %B %Y"
       ),
-      acled_hdx_country = Country |>
+      acled_hdx_location = Country |>
         stringr$str_remove_all("[^[:alnum:]\\s-\\.]") |>
         stringr$str_remove_all("\\.$") |>
         tolower() |>
         stringr$str_replace_all("\\s|\\.", "-"),
```

Comment:
would this serve the same purpose more concisely:

```
stringr$str_replace_all(snakecase$to_snake_case(Country), "_", "-")
```

Reply:
Possibly, but put in a separate PR for review ha!